src/azul/service/elasticsearch_service.py

```diff
@@ -379,11 +373,19 @@ def _generate_paging_dict(self, es_response, pagination, sort_facet):
 
         return page_field
 
-    def page_link(self, search_params, pagination, self_url, sort_facet):
-        return self_url + '?' + urlencode(dict(sort=sort_facet,
+    def page_link(self, pagination, self_url, sort_facet, search_after, search_before) -> Tuple[str, str]:
```

Comment:
I think this is great.

You could probably avoid the duplication by adding a loop within `page_link()`. If you keep the return value as a tuple and remove the duplication with a loop, then the function should probably be called `page_links` instead.

Another option would be to make `search_after` and `search_before` optional arguments, assert exactly one of them is set, then return either next or previous depending on which one it is. I know this is sort of going back on myself, but I like this option better.